packages/components/src/tooltip/src/TooltipTrigger.tsx

```diff
@@ -175,18 +174,13 @@ export function InnerTooltipTrigger({
                     rest,
                     {
                         as,
-                        borderOffset: themeAccessor.getSpace(3),
                         ref: overlayRef,
                         show: isOpen,
                         zIndex
                     }
                 )}
             >
                 {tooltipMarkup}
-                <OverlayArrow
```

Comment:
i feel like the overlay arrow is better located in the tooltip than in the trigger, so i feel like this is a good thing too